.github/workflows/osx_testing.yml

```diff
@@ -18,7 +18,6 @@ jobs:
       fail-fast: false
       matrix:
         runs-on:
-          - macos-10.15
```

Comment:
You can add `macos-12` (beware: `macos-12`, not `macos-12.0`).

Reply:
Done